updatecli/updatecli.d/jdk17.yaml

```
@@ -34,24 +34,18 @@ sources:
           to: _
 
 conditions:
-  checkTemurinAlpineDockerImage:
-    kind: dockerimage
-    name: Check if the container image "eclipse-temurin:<lastVersion>-jdk-alpine" is available
-    disablesourceinput: true
+  checkTemurinAlpineRelease:
+    name: Check if the "<lastVersion>" is available for alpine
+    kind: shell
     spec:
-      image: eclipse-temurin
-      tag: '{{source "lastVersion" }}-jdk-alpine'
-  checkTemurinDebianDockerImages:
-    kind: dockerimage
-    name: Check if the container image "eclipse-temurin:<lastVersion>-jdk-focal" is available
+      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' alpine "x64"
     disablesourceinput: true
+  checkTemurinLinuxRelease:
+    name: Check if the "<lastVersion>" is available for Linux
+    kind: shell
     spec:
-      architectures:
-        - amd64
-        - arm64
-        - s390x
-      image: eclipse-temurin
-      tag: '{{source "lastVersion" }}-jdk-focal'
+      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' standard "x64,aarch64,s390x"
```

In my interactive testing, it worked better for me when I used space separators between the values of the third argument rather than using comma separators.  I inserted debugging print statements and confirmed that I had the correct results when I ran:

```
./jdk-download-url.sh 11.0.23+9 standard "aarch64 x64"
```


```suggestion
      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' standard "x64 aarch64 s390x"
```